pesquini/spec/rails_helper.rb

```diff
@@ -46,5 +46,4 @@
   #
   # The different available types are documented in the features, such as in
   # https://relishapp.com/rspec/rspec-rails/docs
-  config.infer_spec_type_from_file_location!
```

Comment:
Por algum motivo, os testes quebravam por conta dessa linha. Depois de pesquisar, vi que com a retirada dela os testes do rspec voltaram a funcionar.

Date: Sun, 17 May 2015 11:17:54 -0700
From: notifications@github.com
To: pesquini@noreply.github.com
CC: ruandonato@outlook.com
Subject: Re: [pesquini] Pull Request teste_unitario (#6)

In pesquini/spec/rails_helper.rb:

> @@ -46,5 +46,4 @@
>    #
>    # The different available types are documented in the features, such as in
>    # https://relishapp.com/rspec/rspec-rails/docs
> -  config.infer_spec_type_from_file_location!

algum motivo especial para a retirada desta linha?


—
Reply to this email directly or view it on GitHub.